cmd/snap/cmd_unset_test.go

```diff
@@ -45,3 +49,77 @@ func (s *snapSetSuite) TestSnapUnset(c *check.C) {
 	c.Assert(err, check.IsNil)
 	c.Check(s.setConfApiCalls, check.Equals, 1)
 }
+
+func (s *aspectsSuite) TestAspectUnset(c *check.C) {
+	restore := s.mockAspectsFlag(c)
+	defer restore()
+
+	s.mockAspectServer(c, `{"abc":null}`)
+
+	_, err := snapunset.Parser(snapunset.Client()).ParseArgs([]string{"unset", "foo/bar/baz", "abc"})
+	c.Assert(err, check.IsNil)
+}
+
+func (s *aspectsSuite) TestAspectUnsetNoWait(c *check.C) {
+	restore := s.mockAspectsFlag(c)
+	defer restore()
+
+	var reqs int
+	s.RedirectClientToTestServer(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
small nitpick, maybe a `nowait` flag can be passed to `mockAspectServer` so it can be reused here as well?

Reply:
Done, thank you